### PR TITLE
Changed gen-files' encoding to UTF-8 and updated README.md

### LINKED ISSUE #9: Won't build on Windows 10 x64
Spawns next error log:
```
UnicodeEncodeError: 'charmap' codec can't encode character '\xd7' in position 24610: character maps to <undefined>:
  File "C:\Users\Teashrock\Documents\Development\Not-Mine\godot-3.2-stable\SConstruct", line 518:
    SConscript("modules/SCsub")
  File "c:\users\teashrock\envs\py3.7.4\lib\site-packages\scons\SCons\Script\SConscript.py", line 668:
    return method(*args, **kw)
  File "c:\users\teashrock\envs\py3.7.4\lib\site-packages\scons\SCons\Script\SConscript.py", line 605:
    return _SConscript(self.fs, *files, **subst_kw)
  File "c:\users\teashrock\envs\py3.7.4\lib\site-packages\scons\SCons\Script\SConscript.py", line 286:
    exec(compile(scriptdata, scriptname, 'exec'), call_stack[-1].globals)
  File "C:\Users\Teashrock\Documents\Development\Not-Mine\godot-3.2-stable\modules\SCsub", line 17:
    SConscript(x + "/SCsub")
  File "c:\users\teashrock\envs\py3.7.4\lib\site-packages\scons\SCons\Script\SConscript.py", line 668:
    return method(*args, **kw)
  File "c:\users\teashrock\envs\py3.7.4\lib\site-packages\scons\SCons\Script\SConscript.py", line 605:
    return _SConscript(self.fs, *files, **subst_kw)
  File "c:\users\teashrock\envs\py3.7.4\lib\site-packages\scons\SCons\Script\SConscript.py", line 286:
    exec(compile(scriptdata, scriptname, 'exec'), call_stack[-1].globals)
  File "C:\Users\Teashrock\Documents\Development\Not-Mine\godot-3.2-stable\modules\ECMAScript\SCsub", line 63:
    f.write(text.replace('${source}', dump_text_file_to_cpp("misc/godot.d.ts")))
  File "C:\Users\Teashrock\Envs\Py3.7.4\lib\encodings\cp1251.py", line 19:
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
```

### PR DESCRIPTION
**Pull request by Teashrock**
Closes #9. Also changed docs link to 3.2 on added statement about MinGW building.

### CODE REVIEW

File: SCsub [hunk 1]
```diff
@@ -58,10 +58,10 @@ sources = [
 ]
 
 if env['tools']:
-	with open("tools/godot.d.ts.gen.cpp", "w") as f:
+	with open("tools/godot.d.ts.gen.cpp", "w", encoding="utf-8") as f:
```
Comment (Geequlim) [inline SCsub]: Python2 does not support this kind of syntax. Which is the default python version on macOS.

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ It is also possible to replace the ECMAScript engine like V8 or SpiderMonkey but
 ### Compile
 * Clone the source code of [godot](https://github.com/godotengine/godot)
 * Clone this module and put it into `godot/modules/` and make sure the folder name of this module is `ECMAScript`
-* [Recompile godot engine](https://docs.godotengine.org/en/3.0/development/compiling/index.html)
+* [Recompile godot engine](https://docs.godotengine.org/en/3.2/development/compiling/index.html) <b>(Only MinGW is supported on Windows for now!)</b>
```
Comment (Geequlim) [inline README.md]: using `**Bold content**` in markdown instead of  `<b></b>`

File: SCsub (hunk 1 above)
Comment (domis4) [reply SCsub]: Godot 3.2 is now only compileable with Python 3.5+
http://docs.godotengine.org/en/3.2/development/compiling/compiling_for_osx.html

Comment (Teashrock) [reply SCsub:61]: Not only on Mac OS X, other systems too. So, now there is no need to remake that for Python 2.

**Pull request merged**